assets/scripts/index.js

```diff
@@ -0,0 +1,126 @@
+$(document).ready(function(){
+
+  document.querySelector(".js-nav-toggle").addEventListener("click",toggleMenuFunction);
+  document.querySelector(".js-top").addEventListener("click",toTop);
+  window.onscroll = function() {scrollFunction()};
+
+  function toggleMenuFunction() {
+    let burger = document.querySelector(".nav__menu-toggle");
```

Comment:
@edwartexe if you are using some element in the js, you should use a js- prefixed class name. In that way you know the element is associated to a javascript action and avoid losing functionality if for some reason you need to change the class name
```suggestion
    let burger = document.querySelector(".js-nav-menu-toggle");
```